scripts/gen-inject-yaml.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
```

Comment:
Renamed `scripts/ca-bundle.sh` because the primary function of this script is to generate the template.